### PR TITLE
Add README and LICENSE

### PR DESCRIPTION
**Pull request by BlakeWilliams**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,68 @@
...
+For example you could tell the engine to only check very large blocks of code:
+
+```
+engines:
+  duplication:
+    enabled: true
+    config:
+      languages:
+        javascript:
+          mass_threshold: 300
```
Comment (BlakeWilliams) [inline README.md]: This part isn't true yet, it needs to be updated to work with the new config format. Thank you README driven development.

File: README.md [hunk 2]
```diff
@@ -1 +1,68 @@
...
+`codeclimate-duplication` is an engine that wraps [flay] and supports Ruby,
+Python, JavaScript, and PHP. You can run it in the command line using the Code
+Climate CLI, or on our [hosted analysis platform][codeclimate].
+
+[codeclimate]: https://codeclimate.com/dashboard
+
+## Installation
+
+1. If you haven't already, [install the Code Climate CLI][cli]
+2. Run `codeclimate engines:enable eslint`. This command both installs the
```
Comment (GordonDiggs) [inline README.md]: wrong engine name

Comment (BlakeWilliams) [inline README.md]: Oops, copied and pasted and forgot to update it. :stuck_out_tongue:

File: README.md (hunk 1 above)
Comment (GordonDiggs) [inline README.md]: I think we should omit it until it's true

Comment (GordonDiggs) [inline README.md]: Otherwise someone will try it and mention that it doesn't work :)

Comment (BlakeWilliams) [inline README.md]: I was just going to keep this open until that gets fixed. It should be a quick, minor fix.

Comment (GordonDiggs) [inline README.md]: Ah - that's fair

**Comment by GordonDiggs** [comment]
LGTM

**Pull request merged**